docs/sections/calculations/errors.md

```diff
@@ -1,302 +1,68 @@
+Remember that you can run `verdi code list` to check what codes you have available and their information.
 
 ```{code-block} ipython
 
-In [4]: builder.metadata.options.resources = {'num_machines': 1}
-   ...: builder.metadata.options.max_wallclock_seconds = 30 * 60
+In [1]: code_pw = load_code(<CODE_PK>)
+   ...: builder = code_pw.get_builder()
 
 ```
 
-Again, to see the list of available options, type `builder.metadata.options.` and hit the TAB button.
-
-### Preparation of inputs
-
-A Quantum ESPRESSO calculation needs a number of inputs:
-
-1. [Pseudopotentials](<https://en.wikipedia.org/wiki/Pseudopotential>)
-2. a structure
-3. a mesh in reciprocal space (k-points)
-4. a number of input parameters
-
-These are mirrored in the inputs of the `aiida-quantumespresso` plugin (see [documentation](<https://aiida-quantumespresso.readthedocs.io/en/stable/user_guide/calculation_plugins/pw.html>)).
-We'll start with the structure, k-points, and pseudopotentials and leave the input parameters as the last thing to setup.
-
-:::{admonition} Exercise
-
-Use what you learned in the basics section to load the `structure` and `kpoints` inputs for your calculation:
-
-* Use a silicon crystal.
-* Define a `2x2x2` mesh of k-points.
-
-Note: If you just copy and paste code that you executed previously, this may result in duplication of information on your database.
-In fact, you can re-use an existing structure stored in your database [^f1].
-Use a combination of the bash command `verdi data structure list` and the python function `load_node()` to get an object representing the structure created earlier.
-
-:::
-
-### Attaching the input information to the calculation
-
-Once you've created a `structure` node and a `kpoints` node, attach it to the calculation:
+You will need to set the rest of the inputs by yourself, the most critical being `structure`, `kpoints`, `pseudos` and the `resources` options under the builder's `metadata`.
+An incomplete example of how to set these parameters are shown below.
 
 ```{code-block} ipython
 
-In [5]: builder.structure = structure
-   ...: builder.kpoints = kpoints
+In [2]: builder.structure = ...
+   ...: builder.kpoints = ...
+   ...: builder.pseudos = ...
+   ...: builder.metadata.options.resources = {'num_machines': 1}
 
 ```
 
-:::{note}
+:::{attention}
 
-The builder accepts both *stored* and *unstored* data nodes.
-AiiDA will take care of storing the unstored nodes upon submission.
-If you decide not to submit, nothing will be stored in the database.
+The `resources` provided above will work for a locally hosted code (which should tipically be the case for these tutorial), but if you are running these tests in a cluster, you may need to set up account permissions or other options (you can consult the section for [schedulers](https://aiida.readthedocs.io/projects/aiida-core/en/latest/topics/schedulers.html) of the AiiDA documentation).
```

Comment:
```suggestion
The `resources` provided above will work for a locally hosted code, which should typically be the case for this tutorial.
If you are running these tests in a cluster, you may need to set up account permissions or other options (you can consult the section for [schedulers](https://aiida.readthedocs.io/projects/aiida-core/en/latest/topics/schedulers.html) of the AiiDA documentation for further information).
```
In or on a cluster?

Reply:
You are right, it seems to be "on". I hope you don't mind I took the liberty of editing your textbox so that I could just commit this with that correction too.